src/common/libkvs/treeobj.c

```diff
@@ -511,14 +512,13 @@ json_t *treeobj_create_val (const void *data, int len)
     char *xdata;
     json_t *obj = NULL;
 
-    xlen = sodium_base64_encoded_len (len, sodium_base64_VARIANT_ORIGINAL);
+    xlen = base64_encoded_length (len) + 1; /* +1 for NUL */
     if (!(xdata = malloc (xlen))) {
         errno = ENOMEM;
```

Comment:
`malloc()` sets errno (not a prob with this commit, but thought I'd call it out since I noticed).